### PR TITLE
[Relay] docs

### PR DESCRIPTION
**Pull request by cgyurgyik**
Updates and places Relay docs in the book.

### CODE REVIEW

File: docs/frontends/tvm-relay.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-======================
+# TVM Relay
 
-This is an in-progress compiler from [TVM][]'s intermediate representation, [Relay][], targeting Calyx.
+[TVM][] is a compiler for machine learning frameworks that can 
+optimize and target kernels to several different backends.
 
+The TVM Relay frontend lives in the [relay-lang][] folder in the
+Calyx repository and generates Calyx components from the Relay
+intermediate representation.
```
Comment (sampsyo) [inline docs/frontends/tvm-relay.md:8]: Super minor, but I did like the above deleted explanation of what Relay is:

> [TVM][]'s intermediate representation, [Relay][], […]

**Review by sampsyo — COMMENTED** [review]
Excellent! Looks great!

File: docs/frontends/tvm-relay.md (hunk 1 above)
Comment (cgyurgyik) [reply docs/frontends/tvm-relay.md:12]: Updated - I'll let you OK it.

Comment (rachitnigam) [reply docs/frontends/tvm-relay.md:12]: LGTM! Merge it

**Pull request merged**